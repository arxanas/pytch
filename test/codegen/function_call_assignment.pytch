def id(x) => x
let foo = id(1)
print(foo)
