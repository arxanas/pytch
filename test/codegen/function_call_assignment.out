def id(x):
    return x
foo = id(1)
print(foo)
